src/Form/TimesheetEditForm.php

```diff
@@ -72,6 +87,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 },
             ])
             ->add('description', TextareaType::class, [
+                'help' => $this->translator->trans('timesheet.description.help'),
```

Comment:
'help' => 'timesheet.description.help',

Reply:
Translation will automatically happen in the template if you added the translations to message.xx.xliff

Reply:
great. The documentation on symfony website is not very clear about that.